### DISCUSSION TITLE
Midi CC Automation View Customization

### DISCUSSION THREAD

**Discussion by NmI83** [SynthstromAudible/DelugeFirmware#825] [Ideas]
I would appreciate a possibility to customize the Automation View for Midi CC in the following way:

- Automation Overview - deactivate non used CC buttons so that only used/available CCs are lit like in normal Automation View
- maybe Shift + CC button to toggle between lit and non-lit

- Rename CC buttons    - possibility to rename a CC button with the corresponding Effekt/Value for it`s Synth Parameter

With this function we could save a template where every Midi channel got it`s available named CCs. This could really improve the workflow.

**Comment by n3ptun3s** [comment]
Maybe we could implement a system, where we have a folder called "Midi" where you can drop a .txt oder .xml file with a structure like
CC01 = "blabla"
CC74 = Filter Cutoff
etc
for each external midi hardware.

Renaming on the deluge is great, but time consuming.
In this way we could also share the files.
